README.md

```diff
@@ -2,11 +2,11 @@
 
 # Project Name
 
-Pop-up Window.
+Form Validation.
 
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove authors who never contributed to this Pull request from your readme file.